doc/build-openbsd.md

```diff
@@ -76,25 +80,5 @@ Build and run the tests:
 ```bash
 gmake # use -jX here for parallelism
 gmake check
+gmake install
 ```
-
-Resource limits
```

Comment:
Why do you remove this section?

Reply:
Because it only affected builds using GCC.

Reply:
It can affect any kind of build when the build memory size exceeds 1.5GB again for some compilation unit. It is mentioned here to avoid issues being created, by providing a clear workaround.